client/app/lib/styl/tooltip.styl

```diff
@@ -0,0 +1,58 @@
+$lightBg = white
+$lightColor = #727272
+
+div.kdtooltip.is-light
+  width                       250px
+  box-shadow                  0px 2px 10px 0px rgba(0,0,0,0.05)
+  font-size                   14px
+  letter-spacing              0.13px
+  line-height                 20px
+
+  div.wrapper
+    bg                        color, white
+    shadow                    0 10px 20px 5px rgba(0,0,0,0.15)
+    padding                   20px 20px 20px 20px
+
+  .copied-link strong
+    font-weight               600
+    color                     $lightColor
```

Comment:
this color probably needs to be applied to only `.copied-link`, only `font-weight` needs to be applied to `strong` element.